src/NzbDrone.Core/Organizer/FileNameBuilder.cs

```diff
@@ -481,6 +462,17 @@ private string AddAbsoluteNumberingTokens(string pattern, Dictionary<string, Fun
             return pattern;
         }
 
+        private void AddMovieTokens(Dictionary<string, Func<TokenMatch, string>> tokenHandlers, Movie movie)
+        {
+            tokenHandlers["{Movie Title}"] = m => movie.Title;
+            tokenHandlers["{Movie CleanTitle}"] = m => CleanTitle(movie.Title);
+        }
+
+        private void AddReleaseDateTokens(Dictionary<string, Func<TokenMatch, string>> tokenHandlers, int releaseYear)
+        {
+            tokenHandlers["{Release Year}"] = m => string.Format("({0})", releaseYear.ToString()); //Do I need m.CustomFormat?
```

Comment:
We should probably remove the () from the year as it's up to the user if they want them or not.